documentation/guide-i18n.asciidoc

```diff
@@ -33,10 +33,10 @@ The i18n properties file must be called `ApplicationMessages_la_CO.properties` w
 ApplicationMessages_en_US.properties
 [source]
 ----
-waiter=Waiter
-chief=Chief
-cook=Cook
-barkeeper=Barkeeper
+user1=User1
+admin=Admin
+user3=User3
+user2=User2
```

Comment:
IMHO this `use1` to `user3` does not make much sense here.

Reply:
removed